app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/newdevicenotice/NewDeviceNoticeEmailAccessScreen.kt

```diff
@@ -0,0 +1,194 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.PreviewScreenSizes
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.ui.auth.feature.newdevicenotice.NewDeviceNoticeEmailAccessAction.ContinueClick
+import com.x8bit.bitwarden.ui.auth.feature.newdevicenotice.NewDeviceNoticeEmailAccessAction.EmailAccessToggle
+import com.x8bit.bitwarden.ui.auth.feature.newdevicenotice.NewDeviceNoticeEmailAccessEvent.NavigateToTwoFactorOptions
+import com.x8bit.bitwarden.ui.platform.base.util.EventsEffect
+import com.x8bit.bitwarden.ui.platform.components.button.BitwardenFilledButton
+import com.x8bit.bitwarden.ui.platform.components.scaffold.BitwardenScaffold
+import com.x8bit.bitwarden.ui.platform.components.toggle.BitwardenSwitch
+import com.x8bit.bitwarden.ui.platform.components.util.rememberVectorPainter
+import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
+
+/**
+ * The top level composable for the new device notice email access screen.
+ */
+@Composable
+fun NewDeviceNoticeEmailAccessScreen(
+    onNavigateToTwoFactorOptions: () -> Unit,
+    viewModel: NewDeviceNoticeEmailAccessViewModel = hiltViewModel(),
+) {
+    val state by viewModel.stateFlow.collectAsStateWithLifecycle()
+    EventsEffect(viewModel = viewModel) { event ->
+        when (event) {
+            NavigateToTwoFactorOptions -> onNavigateToTwoFactorOptions()
+        }
+    }
+
+    BitwardenScaffold(
+        modifier = Modifier
+            .fillMaxSize(),
+    ) {
+        NewDeviceNoticeEmailAccessContent(
+            email = state.email,
+            isEmailAccessEnabled = state.isEmailAccessEnabled,
+            onEmailAccessToggleChanged = remember(viewModel) {
+                { newState ->
+                    viewModel.trySendAction(EmailAccessToggle(newState = newState))
+                }
+            },
+            onContinueClick = { viewModel.trySendAction(ContinueClick) },
+        )
+    }
+}
+
+@Composable
+private fun NewDeviceNoticeEmailAccessContent(
+    email: String,
+    isEmailAccessEnabled: Boolean,
+    onEmailAccessToggleChanged: (Boolean) -> Unit = {},
+    onContinueClick: () -> Unit = {},
+) {
+    Column(
```

Comment:
Since you went the route of adding a top-level column, be sure to pass in a modifier.

We also do not need default params for the `onEmailAccessToggleChanged` and `onContinueClick`